Polyhedron/demo/Polyhedron/Plugins/Mesh_3/raw_image.ui

```diff
@@ -65,161 +84,191 @@
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="groupBox" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
       <string>Image dimensions</string>
      </property>
-     <layout class="QVBoxLayout" name="verticalLayout_2" >
+     <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
-       <layout class="QGridLayout" >
-        <item row="0" column="0" >
-         <widget class="QLabel" name="label_dim" >
-          <property name="text" >
+       <layout class="QGridLayout">
+        <item row="0" column="0">
+         <widget class="QLabel" name="label_dim">
+          <property name="toolTip">
+           <string>Number of voxel along the axis.</string>
```

Comment:
That one is good. It does provide a better information than the text of the label itself.